src/Components/Pagination.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { PageNumbers } from './PageNumbers';
+import { ShowItems } from './ShowItems';
+
+interface Item {
+  name: string,
+}
+
+type State = {
+  total: Item[],
+  perPage: number,
+  currentPage: number,
+};
+
+class Pagination extends React.Component<{}, State> {
+  state: State = {
+    total: [],
+    perPage: 5,
+    currentPage: 1,
+  };
+
+  componentDidMount() {
+    const array = this.fillTotal();
+
+    this.setState({ total: array });
+  }
+
+  fillTotal = () => {
+    const arrayOfItems = [];
+
+    for (let i = 1; i <= 42; i += 1) {
```

Comment:
```suggestion
    for (let i = 1; i <= 42; i ++) {
```

Reply:
ESLint doesn't allow such short form. It returns "Unary operator '++' used"

Reply:
that's strange. I'll check ESLint config for this task